README.md

```diff
@@ -35,6 +35,48 @@ The following settings must be configured before running the app:
   Sets the hostname required by webpack for building the frontend. Should likely be whatever you set
   the host to in your /etc/hosts or the hostname that you're accessing it from. Likely `od.odl.local`.
 
+#### Keycloak configuration
```

Comment:
I think this is a little confusing as written: If I'm a new team member or even someone just joining MIT Open project, looking at https://github.com/mitodl/mit-open/tree/130-keycloak-integration-logout#configure-required-env-settings, I might thing these are required variables for local development. But they're not, right? I can still log in via django `/admin`. It's also a little confusing where to get the values from (though, maybe this will be easier in the future—thx for sending them to me!).

**Suggestion:** It might make sense to put this under the `## Optional Setup` heading. Additionally, a short sentence describing the feature might be useful.

```
## Optional Setup

### Email Setup
The app is usable without email-sending capability, but there is a lot of app functionality ... 

### Keycloak SSO Setup
*Something here*

env vars
```

Reply:
I agree this is not clear.  These will be required at some point.  There is no process defined for how to get the values and I think it's a bit early to define that process.  It will be done soon and be the same for all OL apps.